modules/tasksUI.js

```diff
@@ -18,104 +21,111 @@ export default class TasksUI {
+          });
         });
     }
   };
 
-  static addTask() {
-    // console.log('event works');
-    const dB = JSON.parse(localStorage.getItem('tasksDB')) || [];
-    // console.log(dB);
-    const addTaskInput = document.querySelector('#task-input');
-    if (addTaskInput.value) {
-      // console.log('input filled');
-      // console.log(addTaskInput.value);
-      // if (localStorage.getItem('tasksDB') === null) {
-      // console.log('database empty');
-      const index = dB.length + 1;
-      const task = new Task(addTaskInput.value, index);
-      const newDB = [...dB, task];
-      localStorage.setItem('tasksDB', JSON.stringify(newDB));
-      TasksUI.checkData();
-      TasksUI.loadTasksUI();
-      /* } else {
-         console.log('database has value');
-        const task = new Task(addTaskInput.value, dB.length);
-        const newDB = [...dB, task];
-        localStorage.setItem('tasksDB', JSON.stringify(newDB));
-        TasksUI.checkData();
-        TasksUI.loadTasksUI();
-       } */
-    }
-  }
-
-  static removeTask(event) {
-    const tasksDbInstance = JSON.parse(localStorage.getItem('tasksDB')) || [];
-    const order = parseInt(event.target.id, 10) - 1;
-    tasksDbInstance.splice(order, 1);
-    localStorage.setItem('tasksDB', JSON.stringify(tasksDbInstance));
-    tasksDbInstance.forEach((element) => {
-      // console.log(element)
-      if (element.index - 1 >= order) {
-        element.index -= 1;
-        // console.log(element.index);
-      }
-    });
-    localStorage.setItem('tasksDB', JSON.stringify(tasksDbInstance));
-    TasksUI.loadTasksUI();
-  }
+  // static removeTask(event) {
+  //   const tasksDbInstance = JSON.parse(localStorage.getItem('tasksDB')) || [];
+  //   const order = parseInt(event.target.id, 10) - 1;
+  //   tasksDbInstance.splice(order, 1);
+  //   localStorage.setItem('tasksDB', JSON.stringify(tasksDbInstance));
+  //   tasksDbInstance.forEach((element) => {
+  //     // console.log(element)
+  //     if (element.index - 1 >= order) {
+  //       element.index -= 1;
+  //       // console.log(element.index);
+  //     }
+  //   });
+  //   localStorage.setItem('tasksDB', JSON.stringify(tasksDbInstance));
+  //   TasksUI.loadTasksUI();
+  // }
 
   static editTask(event) {
+    // console.log('edited');
     const tasksDbInstance = JSON.parse(localStorage.getItem('tasksDB')) || [];
     // console.log(tasksDbInstance);
-    // console.log(event.target);
-    const order = parseInt(event.target.id, 10) - 1;
+    // console.log(event.target.id);
+    // const order = parseInt(event.target.id, 10) - 1;
     // console.log(order);
     // console.log(tasksDbInstance[order]);
-    const elementInput = document.getElementsByClassName('task-title-text')[order];
-    tasksDbInstance[order].description = elementInput.value;
+    const elementInput = document.querySelector(`#${event.target.id}`);
+    const elementID = elementInput.parentElement.id;
+    // console.log(elementID);
```

Comment:
- [OPTIONAL]  KIndly make sure to remove all comments in keep your codebase clean.